docs/superpowers/specs/2026-05-30-capgo-builder-mcp-onboarding-design.html

```
@@ -0,0 +1,628 @@
+    <div class="pipe reveal">
+      <div class="ph"><div class="n">PHASE 0</div><h3>Preflight</h3><p>Capacitor project? app id? API key present? Is a native build even needed (<code>build&nbsp;needed</code> / <code>probe</code>)?</p><span class="badge">mostly auto · 1 gate: login</span><span class="arrow">›</span></div>
+      <div class="ph"><div class="n">PHASE 1</div><h3>App</h3><p>Ensure the app exists in Capgo Cloud; reuse-or-create on conflict.</p><span class="badge">auto · 1 choice</span><span class="arrow">›</span></div>
+      <div class="ph fork"><div class="n">PHASE 2 · forks</div><h3>Credentials</h3><p>The hard part. iOS and Android diverge here — see below.</p><span class="badge">auto + gates + choices</span><span class="arrow">›</span></div>
+      <div class="ph"><div class="n">PHASE 3</div><h3>Build</h3><p>Save creds, then <code>capgo_request_build</code>; poll; surface output URL + QR.</p><span class="badge">auto</span><span class="arrow">›</span></div>
+      <div class="ph"><div class="n">PHASE 4</div><h3>Done</h3><p>Confirm the build, summarize, suggest CI wiring as the next journey.</p><span class="badge">auto</span></div>
+    </div>
+
+    <h3 class="reveal" style="margin-top:46px;font-size:18px;color:var(--soft)">Phase 2 · Credentials — the fork</h3>
+    <div class="tabs reveal">
+      <div class="tabbar" role="tablist">
+        <button role="tab" aria-selected="true" data-tab="ios">🍎 &nbsp;iOS</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)